lib/mapper.js

```diff
@@ -189,14 +205,18 @@ function properties(track){
 
 function traits(identify){
```

Comment:
@sperand-io also wasn't sure if you wanted me to investigate/fix the duplicate trait sending in this PR or in a separate one